src/EnergyPlus/MixedAir.cc

```diff
@@ -3745,6 +3746,16 @@ void OAControllerProps::CalcOAController(EnergyPlusData &state, int const AirLoo
         } else {
             SysSA = curAirLoopFlow.SupFlow;
         }
+
+        // Check if system has a Sizing:System object and a sizing run has been done
+        bool SizingDesRunThisAirSys = false;
+        CheckThisAirSystemForSizing(state, AirLoopNum, SizingDesRunThisAirSys);
+
+        // Get design outdoor air flow rate
+        if (SizingDesRunThisAirSys) {
+            state.dataMixedAir->VentilationMechanical(this->VentMechObjectNum).SysDesOA = state.dataSize->FinalSysSizing(AirLoopNum).DesOutAirVolFlow;
+        }
```

Comment:
This is inside `CalcOAController` so it will fire on every iteration. This needs to move into `GetOAControllerInputs`

Reply:
Sounds good, I'll move it.